p2p/helpers.go

```diff
@@ -94,8 +94,6 @@ func sendMessage(
 		headers = append(headers, resp)
 	}
 
-	duration := time.Since(startTime).Milliseconds()
```

Comment:
very tiny nit but the reason this was here is bc it's immediately after response collection rather than after attempting to close stream - I'm not sure if stream closing could get hung up but putting it at end won't be the *most* accurate measure

Reply:
Close may hang up depending on the implementation but it's not the case with QUIC and yamux, IIRCt. Anyway, I think it is the correct measure. Closing the stream is part of the protocol; the node still waits for it before headers are available. I think the opposite, that it is incorrect to report earlier than when the networking part has finished